config/agents/app/namespace/namespace.yaml

```
@@ -0,0 +1,7 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  creationTimestamp: null
+  name: system
+spec: {}
+status: {}
```

Some redundant fields

```suggestion
metadata:
  name: system
```